src/rgw/rgw_sal_motr.cc

```diff
@@ -3691,7 +3694,27 @@ int MotrMultipartUpload::init(const DoutPrefixProvider *dpp, optional_yield y,
     ent.meta.mtime = ceph::real_clock::now();
     ent.meta.user_data.assign(mpbl.c_str(), mpbl.c_str() + mpbl.length());
     ent.encode(bl);
-
+    std::unique_ptr<RGWObjTags> obj_tags;
+    req_state *s = (req_state *) obj_ctx->get_private();
+    /* handle object tagging */
+    // Verify tags exists and add to attrs
+    auto tag_str_bool = s->info.env->exists("HTTP_X_AMZ_TAGGING");
+    if (tag_str_bool){
```

Comment:
if (s->info.env->exists("HTTP_X_AMZ_TAGGING"){

Reply:
Updated in recent patch